packages/wattpm/lib/utils.js

```diff
@@ -88,16 +88,30 @@ export function getMatchingRuntimeArgs (logger, positional) {
 }
 
 export async function findConfigurationFile (logger, root) {
-  // Find a wattpm.json or watt.json file
-  const configurationFile = await ConfigManager.findConfigFile(root, ['watt.json', 'wattpm.json', 'platformatic.json'])
+  let current = root
+  let configurationFile
+  while (configurationFile === undefined) {
+    // Find a wattpm.json or watt.json file
+    configurationFile = await ConfigManager.findConfigFile(current, ['watt.json', 'wattpm.json', 'platformatic.json'])
```

Comment:
Shall we also account for other variants like YAML, TOML and so forth?

Reply:
Nothing in the current code can handle YAML or TOML, that would need to be done in another PR.